lib/ultrajsonenc.c

```diff
@@ -551,9 +551,20 @@ static void Buffer_AppendIndentUnchecked(JSONObjectEncoder *enc, JSINT32 value)
 {
   int i;
   if (enc->indent > 0)
-    while (value-- > 0)
-      for (i = 0; i < enc->indent; i++)
-        Buffer_AppendCharUnchecked(enc, ' ');
+  {
+    if (enc->indent_chars == NULL)
+    {
+        while (value-- > 0)
+          for (i = 0; i < enc->indent; i++)
+            Buffer_AppendCharUnchecked(enc, ' ');
```

Comment:
Nit: too much indentation, should be two spaces less (also in the `else`).

Reply:
fixed.